internal/gomod/stdlib.go

```diff
@@ -1,21 +1,11 @@
 // THIS FILE IS GENERATED. SEE ./scripts/gen_stdlib_map.sh
-// Generated by: go version go1.16.6 linux/amd64
+// Generated by: go version go1.18.2 darwin/amd64
 package gomod
 
 // isStandardlibPackge determines whether a package is in the standard library
 // or not. At this point, it checks whether the package name is one of those
 // that is found from running "go list std" in the latest released go version.
 func isStandardlibPackge(pkg string) bool {
-	// For certain items in the Universe scope, they have no package.
-	// Because of this, we put them in the "builtin" package, because
-	// that's the name of the file in the go standard library that defines
-	// these items.
-	//
-	// An example would be `error` or `error.Error()`
-	if pkg == "builtin" {
```

Comment:
Is this meant to be removed?

Reply:
This file is auto-generated by `./scripts/gen_stdlib_map.sh`.